libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -323,6 +323,11 @@ public void setContent(CharSequence text) {
         content.fromHtml(text.toString());
         updateFailedMediaList();
         overlayFailedMedia();
+
+        // ToDo:  this is to be removed when feature/async-media is merged
+        // If there are images that are still in progress (because the editor exited before they completed),
+        // set them to failed, so the user can restart them (otherwise they will stay stuck in 'uploading' mode)
+        convertUploadingUponStartToFailed();
```

Comment:
Maybe we can call this `markAllUploadingMediaAsFailed()` like the `EditorFragment` so it's clear this has to do with media?